src/store/getters.ts

```diff
@@ -948,7 +944,7 @@ export const getDraftNameReservation = (state: StateIF): DraftReqI => {
     applicants: [applicant],
     names: nrRequestNames,
     ...nrData,
-    priorityCd: getIsPriorityRequest(state) ? 'Y' : 'N',
+    priorityCd: getPriorityRequest(state) ? PriorityCode.Y : PriorityCode.N,
```

Comment:
ah i see. So you is the `priorityRequest` is true, apply its string code equivalent.

Reply:
Yes.

This was existing code except untyped (as with many other variables, but it's getting better).